crowbar_framework/app/models/network_service.rb

```diff
@@ -413,8 +412,6 @@ def build_net_info(network, name, db = nil)
     db["network"].each { |k,v|
       net_info[k] = v unless v.nil?
     }
-    net_info["usage"]= network
-    net_info["node"] = name
```

Comment:
@vuntz I think you might be missing my point, unless you mean it's already available somewhere else *in this function*?  My point was not that this is needed *now*, but that it will be needed if in the future we do https://trello.com/c/7W51qfqu.